mgmtsystem/ec2.py

```diff
@@ -55,10 +61,30 @@ def __init__(self, **kwargs):
         username = kwargs.get('username')
         password = kwargs.get('password')
 
+        for elb_region in elb_regions():
```

Comment:
I knew about this, I just wanted to make it working and then get it from method. I hope it's okay now.